presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveStorageFormats.java

```diff
@@ -249,11 +256,15 @@ private static void assertSelect(String query, String tableName)
                 .containsExactly(expectedRows);
     }
 
-    private static void setRole(String role)
+    private void setAdminRole()
     {
+        if (!adminRoleEnabled) {
+            return;
+        }
+
```

Comment:
```suggestion
        if (adminRoleEnabled) {
            return;
        }

```

Please invert condition. When `adminRoleEnabled` then there is not need to `setAdminRole` (not inverse).